.github/workflows/test-e2e.yml

```diff
@@ -89,10 +90,10 @@ jobs:
           helm_gcloud_sqlproxy_chart_version: 0.19.7
           helm_oidc_proxy_chart_version: 0.0.22
           helm_imagetag_update: 'ui'
-      - name: wait for status checks
-        uses: broadinstitute/datarepo-actions/actions/main@0.57.0
-        with:
-          actions_subcommand: 'waitfordeployment'
+      - name: "Wait for deployment to come back online"
+        uses: broadinstitute/datarepo-actions/actions/wait-for-deployment@0.59.0
+        env:
+          DEPLOYMENT_TYPE: 'ui'
```

Comment:
I am guessing the other envs don't need to be set since it's the UI?

Reply:
I think I do set this for the other actions - I didn't define a default.

Reply:
But, we don't need to define the "desired githash" for this since we're not waiting for a specific version to deployed. Although, one day we might want to add that.